### PR TITLE
[2/3] Create serializer for ride

### PR DESCRIPTION
**Pull request by katesliang**
## Overview

Create serializer for ride

## Changes Made

- Create serializer for ride
- Create simple serializer for person
- Create serialize for path

## Test Coverage

Tested when creating and getting a ride (in next PR)

## Next Steps

Create routes for creating and getting a ride

### CODE REVIEW

File: src/ride/serializers.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+from ride.models import Ride
+
+
+class RideSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(source="path.id")
+    creator = serializers.SerializerMethodField("get_creator")
+    driver = serializers.SerializerMethodField("get_driver")
+    riders = serializers.SerializerMethodField("get_riders")
+    path = serializers.SerializerMethodField("get_path")
+
```
Comment (alanna-zhou) [inline src/ride/serializers.py]: same comment as the other PR

**Review by alanna-zhou — CHANGES_REQUESTED** [review]
same comment as other PR

**Review by alanna-zhou — APPROVED** [review]
GOAT.

such a fast learner, very quirky.

**Review by kidzegeye — APPROVED** [review]
LGTM

**Review by chalo2000 — APPROVED** [review]
LGTM, I am very happy with the way you are splitting apart these PRs to make them more readable for us :)

File: src/path/serializers.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+class PathSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Path
+        fields = (
+            "id",
+            "start_location_place_id",
+            "start_location_name",
+            "end_location_place_id",
+            "end_location_name",
```
Comment (noah-solomon) [inline src/path/serializers.py:13]: loving the specificity 🤩

**Review by noah-solomon — APPROVED** [review]
valid valid valid

**Pull request merged**